sdk/file_sdk/api_impl/stream_handler.cpp

```diff
@@ -68,14 +66,22 @@ void StreamHandler::SetProtection(const UserPolicy& policy) {
 void StreamHandler::RemoveProtection() {
 }
 
-void StreamHandler::Commit(std::shared_ptr<IStream> outputStream, std::string& outputExtension) {
-  mFileFormat->UpdateProperties(mProperties, {});
-  mFileFormat->Commit(outputStream, outputExtension);
-}
-
 time_t StreamHandler::GetCurrentTime() {
```

Comment:
please change it to use the correct time format like i did, the sample should show the correct way to use the sdk...
Also, it should be UTC time